docs/building-a-form/available-components.md

```diff
@@ -338,7 +374,37 @@ Use this feature to collect multiple items with the same form questions, such as
 
 #### Usage guidelines
 
-In `formConfig`, define this in the data definition.
+In order to display multiple items with the same form questions, the data in the `schema` must be passed as an array, with each group of questions an `item` in that `array`. The `schema` and `uiSchema` for the group of questions within the `items` object is structured the same as other fields.
```

Comment:
The phrase "must be passed as an array" led me to think that it would be a JS array in the schema. How about "must be defined as `type: 'array'`" or something like that?